### PR TITLE
Refactor: Changes in AircraftModel.js

### PR DESCRIPTION
**Pull request by Cedervall**
The purpose of this pull request is to merge changes from seconds to milliseconds in the AircraftModel.js file

### CODE REVIEW

File: src/assets/scripts/client/aircraft/AircraftModel.js [hunk 1]
```diff
@@ -831,9 +831,9 @@ export default class AircraftModel {
         const waypointSpeed = waypointModel.speedMaximum;
         const waypointDistance = this.positionModel.distanceToPosition(waypointModel.positionModel);
         const speedChange = waypointSpeed - this.speed;
-        const decelerationRate = -this.model.rate.decelerate / 2;   // units of rate.decel are 'knots per 2 seconds'
+        const decelerationRate = -this.model.rate.decelerate / 2000;   // units of rate.decel are 'knots per 2 seconds, in milliseconds'
```
Comment (TypAnna) [inline src/assets/scripts/client/aircraft/AircraftModel.js]: Has rate.decel changed units?  Looking in the code base I think decelrate is always given in knots per second, so we should keep the old comment

File: src/assets/scripts/client/aircraft/AircraftModel.js [hunk 2]
```diff
@@ -1753,9 +1753,12 @@ export default class AircraftModel {
         const headingDifference = angle_offset(course, this.heading);
         const bearingFromAircaftToRunway = this.positionModel.bearingToPosition(datum);
         const angleAwayFromLocalizer = course - bearingFromAircaftToRunway;
-        const turnTimeInSeconds = abs(headingDifference) / PERFORMANCE.TURN_RATE;    // time to turn headingDifference degrees
+        const turnTimeInMilliSeconds = abs(headingDifference) / PERFORMANCE.TURN_RATE * TIME.ONE_SECOND_IN_MILLISECONDS;    // time to turn headingDifference degrees
+        // Radians / seconds is standard and will be used as seconds. 
         // TODO: this should be moved to a class method `.getTurningRadius()`
-        const turningRadius = this.speed * (turnTimeInSeconds * TIME.ONE_SECOND_IN_HOURS);  // dist covered in the turn, nm
+        const turningRadius = this.speed * (turnTimeInMilliSeconds * TIME.ONE_HOUR_IN_MILLISECONDS);  // dist covered in the turn, nm
```
Comment (TypAnna) [inline src/assets/scripts/client/aircraft/AircraftModel.js]: Shouldnt it be `this.speed * (turnTimeInMilliSeconds * TIME.ONE_MILLISECOND_IN_HOURS)`?

**Review by TypAnna — CHANGES_REQUESTED** [review]
See comments and change base branch to references-milliseconds. Also merge that branch into yours to resolve conflicts.

File: src/assets/scripts/client/aircraft/AircraftModel.js [hunk 3]
```diff
@@ -2349,7 +2354,8 @@ export default class AircraftModel {
         if (this.target.expedite) {
             climbRate = this.model.rate.climb;
         }
-
+        
+        // TODO: Change climbRate to milliseconds
```
Comment (TypAnna) [inline src/assets/scripts/client/aircraft/AircraftModel.js]: @willwull has handeld this

File: src/assets/scripts/client/aircraft/AircraftModel.js [hunk 4]
```diff
@@ -2324,6 +2328,7 @@ export default class AircraftModel {
             descentRate = this.model.rate.descent;
         }
 
+        // TODO: Change descentRate to milliseconds
```
Comment (TypAnna) [inline src/assets/scripts/client/aircraft/AircraftModel.js]: I have handeld this:)

**Review by cristianbretti — APPROVED** [review]
LGTM!

**Pull request merged**